src/compiler/diagnosticMessages.json

```diff
@@ -195,7 +195,7 @@
         "category": "Error",
         "code": 1063
     },
-    "Ambient enum elements can only have integer literal initializers.": {
+    "In ambient enum declarations member initializer must be constant expression.": {

```

Comment:
nit. "Enum member initializer must be constant expression in an ambient enum declaration"

the emphasis should be on the enum member initializer and not the ambient enum declaration.